static/app/views/issueDetails/header.tsx

```diff
@@ -57,11 +57,13 @@ function GroupHeaderTabs({
   project,
 }: GroupHeaderTabsProps) {
   const organization = useOrganization();
+  const location = useLocation();
 
   const {getReplayCountForIssue} = useReplayCountForIssues({
     statsPeriod: '90d',
   });
   const replaysCount = getReplayCountForIssue(group.id, group.issueCategory);
+  const queryParams = omit(location.query, ['sort']);
```

Comment:
I don't see any issues with omitting sort from the query params, but a comment here explaining why this is necessary would be great!